src/pages/error/error.jsx

```diff
@@ -0,0 +1,32 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash" className="error">
+        <div id="masthead">
+          <div className="inner">
+            <div id="header">
+              <a href="#/"><img src="./img/logo.png" width="200" height="53" alt="Webmaker" /></a>
```

Comment:
this looks a little hardcody, where are these width/height values coming from? Can we make those simply based on the image we link to?

Secondary thought: does it make sense to make the masthead a reusable component?